src/inbox/inbox.service.ts

```diff
@@ -56,6 +120,22 @@ export class InboxService {
     return Message.from(data);
   }
 
+  private async loadThumbnail(hash: string): Promise<string | undefined> {
+    try {
+      const thumbnailBuffer = await this.thumbnail_bucket.get(hash);
+      const base64 = thumbnailBuffer.toString('base64');
+      const mimeType = 'image/webp';
```

Comment:
Thumbnail isn't always an `image/webp` right? The mime type should either be set in the bucket OR the file extension should be used.